ChelseasFieldJournal/src/GUI/GuiDialogToolJournal.cs

```diff
@@ -0,0 +1,202 @@
+﻿using Cairo;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Vintagestory.API.Client;
+using Vintagestory.API.Common;
+using Vintagestory.API.Common.Entities;
+using Vintagestory.API.Config;
+using Vintagestory.API.MathTools;
+using static Ele.ChelseasFieldJournal.ModConstants;
+
+namespace Ele.ChelseasFieldJournal
+{
+    public class GuiDialogToolJournal : GuiDialogGeneric
+    {
+        
+        private int leftPageNumber { get; set; }
```

Comment:
Explain to me why we need to know the left page number and right page number and cannot calculate it from the Chapter index

Reply:
Chapter Index is the current chapter, i.e. chapter 1 = advice, 2 = ore, 3 = fauna, etc. ChapterLeftPage is intended to be the chapter-specific page. I'll make it calculated just off ChapterLeftPage tho.